client/json-rpc/src/response.rs

```diff
@@ -197,3 +202,13 @@ impl ResponseAsView for AccountStateWithProofView {
         }
     }
 }
+
+impl ResponseAsView for TransactionsWithProofsView {
+    fn from_response(response: JsonRpcResponse) -> Result<Self> {
+        if let JsonRpcResponse::TransactionsWithProofsResponse(resp) = response {
+            Ok(resp)
+        } else {
+            Self::unexpected_response_error::<Self>(response)
+        }
+    }
+}
```

Comment:
could you remove changes to this response.rs file? I don't plan to continue maintain this client, I don't think we need add more to here, if someone uses it, it will increase migration cost.